ftw/mobile/buttons.py

```diff
@@ -99,3 +102,33 @@ def endpoint(self):
     def startup_cachekey(self):
         return (self.context.restrictedTraverse('@@mobilenav')
                 .get_startup_cachekey())
+
+
+class MultilanguageButton(BaseButton):
+
+    def label(self):
+        context = self.context.aq_inner
+        portal_state = getMultiAdapter(
+            (context, self.request),
+            name=u'plone_portal_state')
+        current_language = portal_state.language()
+        return current_language
+
+    def position(self):
+        return 300
+
+    def data(self):
+        portal_languages = self.context.portal_languages
+
+        langs = portal_languages.getAvailableLanguages()
```

Comment:
@jone, I have pushed a commit onto this branch which might be the solution to your question. Could you please review it?